lib/web/apiserver.go

```diff
@@ -410,7 +410,8 @@ func NewHandler(cfg Config, opts ...HandlerOption) (*RewritingHandler, error) {
 
 	// Desktop access endpoints.
 	h.GET("/webapi/sites/:site/desktops", h.WithClusterAuth(h.getDesktopsHandle))
-	h.GET("/webapi/sites/:site/desktop/:desktopUUID/connect", h.WithClusterAuth(h.handleDesktopAccessWebsocket))
+	h.GET("/webapi/sites/:site/desktops/:desktopUUID", h.WithClusterAuth(h.getDesktopHandle))
```

Comment:
`desktopUUID` -> `desktopID`

Reply:
I actually have a change in progress to rename this to `desktopName`. It's not a UUID.